docs/minio-limits.md

```diff
@@ -1,56 +1,64 @@
 |Read quorum| N/2|
-|Write quorum| N/2+1|
+|Write quorum| N/2 + 1|
 
-### Browser Access
+### 2. <a name="maximum-browser-upload-size"></a>Maximum Browser Upload Size 
 
-|Item|Specification|
+|**Item**|**Limit**|
 |:---|:---|
-|Web browser upload size limit| 5 TiB|
+|Web browser upload size limit| 5 TB|
 
-### Limits of S3 API
+### 3. <a name="maximum-sizes-using-the-s3-api"></a>Maximum Sizes Using the S3 API
 
-|Item|Specification|
+|**Item**|**Limit**|
 |:---|:---|
 |Maximum number of buckets| no-limit|
 |Maximum number of objects per bucket| no-limit|
-|Maximum object size| 5 TiB|
+|Maximum object size| 5 TB|
 |Minimum object size| 0 B|
-|Maximum object size per PUT operation| 5 TiB|
+|Maximum object size per PUT operation| 5 TB|
 |Maximum number of parts per upload| 	10,000|
-|Part size|5 MiB to 5 GiB. Last part can be 0 B to 5 GiB|
-|Maximum number of parts returned per list parts request| 1000|
-|Maximum number of objects returned per list objects request| 1000|
-|Maximum number of multipart uploads returned per list multipart uploads request| 1000|
-
-### List of Amazon S3 API's not supported on Minio
-We found the following APIs to be redundant or less useful outside of AWS S3. If you have a different view on any of the APIs we missed, please open a [github issue](https://github.com/minio/minio/issues).
-
-#### List of Amazon S3 Bucket API's not supported on Minio
-
-- BucketACL (Use [bucket policies](https://docs.minio.io/docs/minio-client-complete-guide#policy) instead)
-- BucketCORS (CORS enabled by default on all buckets for all HTTP verbs)
-- BucketLifecycle (Not required for Minio erasure coded backend)
-- BucketReplication (Use [`mc mirror`](https://docs.minio.io/docs/minio-client-complete-guide#mirror) instead)
-- BucketVersions, BucketVersioning (Use [`s3git`](https://github.com/s3git/s3git))
-- BucketWebsite (Use [`caddy`](https://github.com/mholt/caddy) or [`nginx`](https://www.nginx.com/resources/wiki/))
-- BucketAnalytics, BucketMetrics, BucketLogging (Use [bucket notification](https://docs.minio.io/docs/minio-client-complete-guide#events) APIs)
-- BucketRequestPayment
-- BucketTagging
-
-#### List of Amazon S3 Object API's not supported on Minio
-
-- ObjectACL (Use [bucket policies](https://docs.minio.io/docs/minio-client-complete-guide#policy) instead)
-- ObjectTorrent
-- ObjectVersions
-
-### Object name restrictions on Minio
-Object names that contain characters `^*|\/&";` are unsupported on Windows and other file systems which do not support filenames with these characters. Note that this list is not exhaustive, and depends on the maintainers of the filesystem itself.
+|Part size|5 MB to 5 GB. The last part can be 0 B to 5 GB.|
+|Maximum number of parts returned per list in a request for parts| 1000|
+|Maximum number of objects returned per list in a request for objects| 1000|
+|Maximum number of multi-part uploads returned per list in a request for multi-part uploads| 1000|
+
```

Comment:
Discussed here in the office. I think we need to stick to MiB/GiB/TiB to be technically correct. However this could be perhaps relaxed for ease of readability.